### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by psijkof**
Added an example on how to use fragments to query related content items

### CODE REVIEW

**Comment by agriffard** [comment]
Should we also include OrchardCore.Apis.GraphQL.Abstractions/README.md in mkdocs.yml to make it appear in the docs site?

File: src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md [hunk 1]
```diff
@@ -157,9 +157,117 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
+## Querying related content items
+
+One of the features of Content Items, is that they can be related to other Content Items.
+
+Imagine we have the following Content Types: (abbreviated for readability)
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md]: You don't need to provide some json, just mention types that have Content Picker fields.

File: src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md [hunk 2]
```diff
@@ -157,9 +157,117 @@ Shown in the example above, we have an autoroutePart argument, this is registere
...
+        }
+      ]
+    }
+
+```
+Note that the field ```FavoriteMovies``` is of type ```ContentPickerField``` which allows you select multiple ´Movies´ as per the ```DisplayedContentTypes```
+
+### Get the related content items GraphQL query
+
+Now, if we would want to get the Favorite Movies of the Person items we query, the following query will throw an error:
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md]: I don't think you need a "bad" example.

File: src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md [hunk 3]
```diff
@@ -157,9 +157,117 @@ Shown in the example above, we have an autoroutePart argument, this is registere
...
+          name
+          releaseYear
+        }
+      }
+    }
+  }
+}
+```
+
+**Notice** the ```... on Movie``` fragment, that tells the GraphQL parser to treat the discovered object as ´Movie´.
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md]: This explanation should come before the example.

**Comment by sebastienros** [comment]
Yes, please add this file to the TOC.

File: src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md (hunk 3 above)
Comment (psijkof) [reply src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md]: done

**Comment by agriffard** [comment]
@psijkof : Please add a line like this below the GraphQL link in mkdocs.yml, line 66:
`        - GraphQL queries: OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md`

**Pull request merged**